exercises/concept/the-farm/the_farm_test.go

```diff
@@ -14,192 +14,141 @@ func (wf testWeightFodder) FodderAmount() (float64, error) {
+			cows:             2,
+			wantFodderAmount: 0,
+			wantFodderErr:    NonScaleError,
 		},
 		{
-			description:             "Scale returns 10 with ErrScaleMalfunction for 2 cows",
-			weightFodder:            testWeightFodder{fodder: 10, err: ErrScaleMalfunction},
-			weightFodderDescription: "10 fodder, ErrScaleMalfunction",
-			cows:                    2,
-			wantAmount:              10,
-			wantErr:                 nil,
+			description:      "10 fodder for 2 cows with malfunction error",
+			fodderAmount:     testWeightFodder{fodder: 10, err: ErrScaleMalfunction},
+			cows:             2,
+			wantFodderAmount: 10,
+			wantFodderErr:    nil,
 		},
 		{
-			description:             "Scale returns 1 with ErrScaleMalfunction for 10 cows",
-			weightFodder:            testWeightFodder{fodder: 5, err: ErrScaleMalfunction},
-			weightFodderDescription: "5 fodder, ErrScaleMalfunction",
-			cows:                    10,
-			wantAmount:              1,
-			wantErr:                 nil,
+			description:      "5 fodder for 10 cows with malfunction error",
+			fodderAmount:     testWeightFodder{fodder: 5, err: ErrScaleMalfunction},
+			cows:             10,
+			wantFodderAmount: 1,
+			wantFodderErr:    nil,
 		},
 		{
-			description:             "Negative fodder",
-			weightFodder:            testWeightFodder{fodder: -1, err: nil},
-			weightFodderDescription: "-1 fodder, no error",
-			cows:                    2,
-			wantAmount:              0,
-			wantErr:                 errors.New("negative fodder"),
+			description:      "Negative fodder for 2 cows with no error",
+			fodderAmount:     testWeightFodder{fodder: -1, err: nil},
+			cows:             2,
+			wantFodderAmount: 0,
+			wantFodderErr:    errors.New("negative fodder"),
 		},
 		{
-			description:             "Negative fodder with ScaleError",
-			weightFodder:            testWeightFodder{fodder: -1, err: ErrScaleMalfunction},
-			weightFodderDescription: "-1 fodder, ScaleError",
-			cows:                    2,
-			wantAmount:              0,
-			wantErr:                 errors.New("negative fodder"),
+			description:      "Negative fodder for 2 cows with malfunction error",
+			fodderAmount:     testWeightFodder{fodder: -1, err: ErrScaleMalfunction},
+			cows:             2,
+			wantFodderAmount: 0,
+			wantFodderErr:    errors.New("negative fodder"),
 		},
 		{
-			description:             "Zero cows",
-			weightFodder:            testWeightFodder{fodder: 100, err: nil},
-			weightFodderDescription: "100 fodder, no error",
-			cows:                    0,
-			wantAmount:              0,
-			wantErr:                 errors.New("division by zero"),
+			description:      "100 fodder for zero cows with no error",
+			fodderAmount:     testWeightFodder{fodder: 100, err: nil},
+			cows:             0,
+			wantFodderAmount: 0,
+			wantFodderErr:    errors.New("division by zero"),
 		},
 	}
+
 	for _, test := range tests {
-		t.Run(test.description, func(t *testing.T) {
```

Comment:
Why should t.Run (a good practice in Go) be removed in favor of a self-written t.Logf?

Reply:
Running subtests with `t.Run` is a good practice. Its also a good practice to log tests and subtests. 

I can change the tests to have subtests only (`t.Run`)